### PR TITLE
✨ Create initial README.md for DESOSA'17

### PR DESCRIPTION
**Pull request by sandervdo**
TODO: 

- The CC-BY image is still missing in the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+The systems are from a wide range of domains, including testing frameworks ([Mockito], [JUnit5]), editors ([Neovim], [VSCode]), music ([Beets]), and visualisation ([Kibana]).
+
+[mockito]: http://site.mockito.org/
+[junit5]: http://junit.org/junit5/
+[neovim]: https://neovim.io/
+[vscode]: https://code.visualstudio.com/
+[beets]: http://beets.io/
+[kibana]: https://www.elastic.co/products/kibana
+
+During a 8 week period, the students spent one third of their time on this course, and engaged with these systems in order to understand and describe their software architecture.
```
Comment (vm00z) [inline README.md]: 8-week

File: README.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+[development]: http://www.viewpoints-and-perspectives.info/home/viewpoints/
+
+A third recurring theme is _technical debt_. Large and long existing projects are commonly vulnerable to debt.
+The students assessed the current debt in the systems and provided proposals on resolving this debt where possible.
+
+## First-Hand Experience
+
+```
+TODO: Write this section. Collect all PRs and write a nice summary.
```
Comment (vm00z) [inline README.md:65]: If we do decide to write this contributions chapter, perhaps we can link it here?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+[Rails 24198]: https://github.com/rails/rails/pull/24198
+[Sonic-Pi 1054]: https://github.com/samaaron/sonic-pi/pull/1054
+[OpenCV 4375]: https://github.com/Itseez/opencv/issues/4375
+[OpenTripPlanner 2232]: https://github.com/opentripplanner/OpenTripPlanner/pull/2232
+
+Through these contributions the students often interacted with lead developers and architects of the systems under study, gaining first-hand experience with the architectural trade-offs made in these systems.
+
+## Feedback
+
+While we worked hard on the chapters to the best of our abilities, there will be plenty of omissions and inaccuracies.
```
Comment (vm00z) [inline README.md]: Do we really want to say this like that?

**Review by vm00z — COMMENTED** [review]
Nitpicky changes

**Comment by sandervdo** [comment]
@ValMai I've changed the `README.md` according to your comments. Can you have another look? 😄

**Review by vm00z — APPROVED** [review]
Appoofed

**Pull request merged**